examples/jupyter/nuclear-data-resonance-covariance.ipynb

```diff
@@ -0,0 +1,374 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x14c9e4583668>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.imshow(covariance)\n",
+    "plt.colorbar()"
```

Comment:
You might want to use a [diverging colormap](https://matplotlib.org/examples/color/colormaps_reference.html) with the min/max values set to be negatives of each other (e.g., -0.008 and 0.008) so that it is obvious where zero is.